tests/KeyFixTest.php

```diff
@@ -69,26 +70,9 @@ public function testNoBadChars() {
          $this->assertSame($fixed, $original);
       }
    }
-      
-   public function testDefaultBadChars() {
-      $memoryCache = new TestEphemeral();
-      $cache = new Matryoshka\KeyFix($memoryCache, 40);
-
-      // Empty string and Newline are the default characters this backend fixes.
-      foreach([' ', "\n"] as $badChar) {
-         list($key) = $this->getRandomKeyValue();
-         $badKey = $key . $badChar;
-
-         $cache->set($badKey, $badKey);
-      }
-
-      foreach ($memoryCache->getCache() as $fixed => $original) {
-         $this->assertNotSame($fixed, $original);
-      }
-   }
 
    public function testCustomBadChars() {
-      $badChar = 'a';
+      $badChar = '[a]';
 
       $memoryCache = new TestEphemeral();
       $cache = new Matryoshka\KeyFix($memoryCache, 40, $badChar);
```

Comment:
I would think this should throw an exception.... This isn't a valid regex in php. It would need to start and end with the same char.